demos/speech_recognition_demo/python/utils/deep_speech_seq_pipeline.py

```diff
@@ -0,0 +1,50 @@
+#
+# Copyright (C) 2019-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file is based in part on deepspeech_openvino_0.5.py by Feng Yen-Chang at
+# https://github.com/openvinotoolkit/open_model_zoo/pull/419, commit 529805d011d9b405f142b2b40f4d202bd403a4f1 on Sep 19, 2019.
+#
+from copy import deepcopy
+
+from utils.audio_features import AudioFeaturesSeqPipelineStage
+from utils.rnn_seq_pipeline import RnnSeqPipelineStage
+from utils.ctc_decoder_seq_pipeline import CtcDecoderSeqPipelineStage
+
+
+class DeepSpeechSeqPipeline:
+    def __init__(self, model, lm=None, beam_width=500, max_candidates=None,
+            profile=None, ie=None, device='CPU', online_decoding=False):
+        """
+            Args:
+        model (str), filename of IE IR .xml file of the network
+        lm (str), filename of LM (language model)
+        beam_width (int), the number of prefix candidates to retain during decoding in beam search (default 500)
+        max_candidates (int), limit the number of returned candidates; None = do not limit (default None)
+        profile (dict): a dict with pre/post-processing parameters, see profiles.py
+        ie (IECore or None), IECore object to run NN inference with.  Default is to use ie_core_singleton module.
```

Comment:
We are tending IECore to always be provided from `main()`. For example https://github.com/openvinotoolkit/open_model_zoo/blob/9ca5dbeff80464bf5728e8be25daedfe9a9208d7/demos/common/python/pipelines/async_pipeline.py#L23 Please, do the same.

Reply:
Done, now `ie` is the second arg (next after `profile`).